readme.md

```diff
@@ -61,6 +61,13 @@ Type: `Function`
 
 Function that gets called anytime the object changes.
 
+The function receives three arguments: 
+1. A path to the value that was changed.
```

Comment:
Can also document the format of this "path" here?

Reply:
Will do.